src/jupyterlabcontroller/models/v1/event.py

```diff
@@ -1,24 +1,24 @@
 """Event model for jupyterlab-controller."""
 
-from enum import auto
+from enum import Enum
 
-from safir.pydantic import CamelCaseModel
+from pydantic import BaseModel
 from sse_starlette import ServerSentEvent
 
-from ..enums import NubladoEnum
-
 """GET /nublado/spawner/v1/labs/username/events"""
 
 
-class EventTypes(NubladoEnum):
-    COMPLETE = auto()
-    ERROR = auto()
-    FAILED = auto()
-    INFO = auto()
-    PROGRESS = auto()
+class EventTypes(Enum):
```

Comment:
If we're using Python 3.11 we could use StrEnum instead, and then we could use auto() and still get the right behavior.